x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -80,6 +80,11 @@ export const APP_MANAGEMENT_PATH = `${APP_PATH}/administration`;
 
 export const DETECTIONS_SUB_PLUGIN_ID = `${APP_ID}:${SecurityPageName.detections}`;
 
+/**
+ * The ID of the cases plugin to make it easier for the failed permissions cases to navigate back to the main app
+ */
+export const CASES_SUB_PLUGIN_ID = `${APP_ID}:${SecurityPageName.case}`;
```

Comment:
ahhh but you told me to [make one here](https://github.com/elastic/kibana/pull/102103#discussion_r651974097)
![img](https://media2.giphy.com/media/XKoJJhJZa80mI/giphy.gif?cid=ecf05e47mm7qbtk42qsljpw5roityzmxtmm6inh4ln739ya5&rid=giphy.gif&ct=g)

Reply:
haha sorry, didn't mean it as double the work, I figured your PR would probably get merged before mine so I would just deal with the conflicts. Although @michaelolo24 told me today that we probably don't need them at all once the nav changes hit so maybe it's a mistake 😬

Reply:
![o](https://media3.giphy.com/media/WS6CDvvyNDrhZRFBtT/giphy.gif?cid=ecf05e472mjv6ck0rzjkjbgn0amcnnt6qvlagqwb2wlgz3tb&rid=giphy.gif&ct=g)
im a bit gif happy today